apps/examples/iperf2/README.txt

```diff
@@ -0,0 +1,43 @@
+examples/helloxx
```

Comment:
Do we need this README file? Or it would be better to modify the contents.

Reply:
Thanks I'll fix